src/message.rs

```diff
@@ -72,6 +72,7 @@ pub enum Message {
     Order(MessageKind),
     Dispute(MessageKind),
     CantDo(MessageKind),
+    Rate(MessageKind),
```

Comment:
I'm not sure if we should have a Rate message because rates haven't their own id, rates are just information related to orders, I think rate information are still Order messages with `Action::RateUser`